gcviewer/urls

```diff
@@ -0,0 +1 @@
+http://sourceforge.net/projects/gcviewer/files/gcviewer-${version}.${ext}
```

Comment:
mhm. Should be `https` instead. TLS should be standard and prevents man-in-the-middle attacks.